### ISSUE TITLE
Phone Number field Label positions

### ISSUE THREAD

**Issue by am0** [verbb/formie#190]
**Description**
The subfield label position option affects the main field label position when the "Show Country Code Dropdown" option is enabled.

**Steps to reproduce**
1. Create a phone number field and check the "Show Country Code Dropdown" option
2. Under the 'Appearance' tab, the "Label Position" will always take on the setting from the "Subfield Label Position". For example:

With the subfield label option set to hidden, the field label will also always be hidden.

![Screenshot 2020-10-22 at 12 04 41](https://user-images.githubusercontent.com/618467/96864324-224fc080-1460-11eb-8b0b-b9fe4910ba00.png)

![Screenshot 2020-10-22 at 12 04 59](https://user-images.githubusercontent.com/618467/96864620-9ab68180-1460-11eb-8537-1dbbbac603cf.png)

With the subfield label option set to "above" and the field label set to something hidden, the field label won't hide. Also, if you select a different label position such as right or left, when rendered out it changes slightly but doesn't look right. Some css classes on the container do change- for example, `fui-label-left-input fui-subfield-label-right-input` are added, but the position isn't correct.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: Tested on "dev-craft-3 as 1.2.24" and rolled back to 1.2.24, seems unchanged
- Craft version: 3.5.14
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.24",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.2.25](https://github.com/verbb/formie/releases/tag/1.2.25)
